.github/workflows/e2e-pr.yml

```
@@ -0,0 +1,112 @@
+name: SalesforceCommerceCloud/pwa-kit/e2e-pr
+on:
+    workflow_dispatch:
```

Do we want to add as input the node version so we can update the default node MRT in one place instead of multiple places?

Do we also want a way to skip this workflow as escape hatch in case we need it?

Something along these lines:

```suggestion
    workflow_dispatch:
        inputs:
            skip_workflow:
                description: 'Skip the entire workflow'
                required: false
                default: false
                type: boolean
            node_version:
                description: 'Node version to test with'
                required: false
                default: '22'
                type: string
```

---

Yeah. Good call. I'll add Node ENV as a github variable. so we don't need to make code changes to update node version. I'll also include the skip workflow as an input.

Thanks!